tests/functional/firefox/test_accounts.py

```diff
@@ -7,17 +7,19 @@
 from pages.firefox.accounts import FirefoxAccountsPage
 
 
-@pytest.mark.skip_if_firefox(reason='Download button is only displayed to non-Firefox users')
+# test Firefox state
+@pytest.mark.skip_if_not_firefox(reason='Non-Firefox user does not see Firefox state')
 @pytest.mark.nondestructive
-def test_download_button_displayed(base_url, selenium):
+def test_firefox_state(base_url, selenium):
```

Comment:
Try and keep test names descriptive. If this fails in CI, does it give a good indication of what might be wrong? I'd suggest keepoing this something along the lines of `test_download_buttons_displayed`